.github/workflows/integration_test.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Setup VAL containers
         shell: bash
         env:
+          KDB_OPT: "--vss vss_release_3.1.1.json"
```

Comment:
Is the file present along with 4.0 release in databroker image?
Maybe we should choose different VSS paths that are not changed

Reply:
Yes, at the moment we have both 3.1.1 and 4.0 mentioned in https://github.com/eclipse/kuksa.val/blob/master/kuksa_databroker/Dockerfile.

We need this line to not get "latest VSS version" which is the Databroker default. The regression causing kuksa.val.services builds to fail is that (some tests in) kuksa.val.services use databroker:master, so when kuksa.val Databroker changed to a different VSS version the tests started to fail. A better approach would be if kuksa.val.services woul use an explicit version on Databroker, but so far we do not really have any good release-process for KUKSA.val Databroker